src/data/int/cast.lean

```diff
@@ -157,6 +157,10 @@ by by_cases b ≤ a; simp [h, max]
   ((abs q : ℤ) : α) = abs q :=
 by simp [abs]
 
+lemma coe_int_dvd [comm_ring α] (m n : ℤ) (h : m ∣ n) :
+  (m : α) ∣ n :=
```

Comment:
```suggestion
  (m : α) ∣ (n : α) :=
```
I know Lean inserts this coercion itself, but I think it helps readability? No problem if you prefer not.